src/generator.cs

```diff
@@ -572,6 +589,17 @@ public MemberInformation (IMemberGatherer gather, PropertyInfo pi, Type type, bo
 			is_virtual_method = false;
 		else
 			is_virtual_method = !is_static;
+
+		// Properties can have WrapAttribute on getter/setter so we need to check for this
+		// but only if no Export is already found on property level.
+		if (export is null) {
+			wpmi = new WrapPropMemberInformation (pi);
+			has_inner_wrap_attribute = wpmi.HasWrapOnGetter || wpmi.HasWrapOnSetter;
+
+			// Wrap can only be used either at property level or getter/setter level at a given time.
+			if (wrap_method != null && has_inner_wrap_attribute)
+				throw new BindingException (1063, true, $"The 'WrapAttribute' can only be used either at property level or at getter/setter level in a given time. Property: '{pi.DeclaringType}.{pi.Name}'");
```

Comment:
minor: sounds weird (e.g. both `only` and `either`, also I think it's `at a given time`)
c.c. @chamons as a native english speaker :)

Reply:
I think if you use only you don't need either:

You need only X or Y
vs
You need only either X or Y.

So vote to strike to something like:

 +				throw new BindingException (1063, true, $"The 'WrapAttribute' can only be used at the property or at getter/setter level at a given time. Property: '{pi.DeclaringType}.{pi.Name}'");